packages/cli/lib/developerTestAccountCreate.js

```diff
@@ -0,0 +1,194 @@
+const SpinniesManager = require('./ui/SpinniesManager');
+const {
+  getAccountId,
+  accountNameExistsInConfig,
+  updateAccountConfig,
+  writeConfig,
+} = require('@hubspot/local-dev-lib/config');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  createDeveloperTestAccount,
+} = require('@hubspot/local-dev-lib/developerTestAccounts');
+const { i18n } = require('./lang');
+const {
+  debugErrorAndContext,
+  logErrorInstance,
+} = require('./errorHandlers/standardErrors');
+const {
+  isMissingScopeError,
+  isSpecifiedError,
+} = require('@hubspot/local-dev-lib/errors/apiErrors');
+const {
+  getAccessToken,
+  updateConfigWithAccessToken,
+} = require('@hubspot/local-dev-lib/personalAccessKey');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
+const { uiAccountDescription } = require('./ui');
+const {
+  personalAccessKeyPrompt,
+} = require('./prompts/personalAccessKeyPrompt');
+const { enterAccountNamePrompt } = require('./prompts/enterAccountNamePrompt');
+
+const i18nKey = 'cli.lib.developerTestAccount';
+
+const saveDevTestAccountToConfig = async (env, result, force = false) => {
+  let personalAccessKey = result.personalAccessKey;
+  if (!personalAccessKey) {
+    const configData = await personalAccessKeyPrompt({
+      env,
+      account: result.id,
+    });
+    personalAccessKey = configData.personalAccessKey;
+  }
+
+  let updatedConfig;
+  try {
+    const token = await getAccessToken(personalAccessKey, env);
+    updatedConfig = await updateConfigWithAccessToken(
+      token,
+      personalAccessKey,
+      env
+    );
+  } catch (e) {
+    logErrorInstance(e);
+  }
+  if (!updatedConfig) {
+    throw new Error('Failed to update config with personal access key.');
```

Comment:
Might be worth trying to incorporate this into the error logged on line 53. Looks like the only place `saveDevTestAccountToConfig` this error is being caught and logged

Reply:
yeah probably very rare for that to fail, I'll just move the error up and get rid of this undefined block